### ISSUE TITLE
Tooltip omits last line

### ISSUE THREAD

**Issue by JamesBremner** [cnjinhao/nana#365]
```
int main()
{
    nana::form fm;
    nana::button btn(fm,nana::rectangle{10,10,100,30});
    btn.caption("test");
    btn.tooltip("line1\nline2\nline3");
    fm.show();
    nana::exec();
}
```

Produces

![image](https://user-images.githubusercontent.com/2046227/49882949-a6b27080-fdff-11e8-9b32-99e454162785.png)

Notice also the extra whitespace after the first line.  This is present even when there is only one line, making the tooltip pop-up larger than required.

For more discussion see http://nanapro.org/en-us/forum/index.php?u=/topic/1072/ggtooltip-sizing

**Comment by cnjinhao** [comment]
Thank you for reporting this issue. A fix has been checked into the `hotfix` branch.

**Comment by JamesBremner** [comment]
Why has this issue been closed?  AFAIK there has not been a new release, so the fix is not available for testing and use by ordinary users.  IMHO an issue should not be closed until has been tested in a release.

I have been promising my clients this fix for weeks now.  Why has there not been a new release for so long - previously incremental release came out every month or so.

**Comment by cnjinhao** [comment]
The fix has been checked into the `develop-1.7` for a while. Don't worry, the test is OK, and new version 1.7 is going to be released.
Thank you for your patient.

**Comment by JamesBremner** [comment]
So, version 1.7.0 is finally released and, yes, the tooltips are now properly formatted

![image](https://user-images.githubusercontent.com/2046227/54168267-10183e80-4444-11e9-97da-37c0e32a5512.png)

As it turns out, this issue was only closed 8 days prematurely!
